codejs.js

```diff
@@ -1,91 +1,86 @@
-
-
-const cabanas = document.querySelectorAll (".cabana");
-const poput = document.querySelector('.back-Ground');
-const Xboton = document.querySelector('.Xboton');
+const cabanas = document.querySelectorAll(".cabana");
+const poput = document.querySelector(".back-Ground");
+const Xboton = document.querySelector(".Xboton");
```

Comment:
nit: const in JS should be camel case
```suggestion
const xButton = document.querySelector(".Xboton");
```